test/jdk/javax/xml/crypto/dsig/SecureValidationSystemProperty.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @bug 8301260
+ * @summary Check that secureValidation system property works correctly
+ * @library /test/lib
+ * @run main/othervm -Dorg.jcp.xml.dsig.secureValidation=true SecureValidationSystemProperty
+ */
+
+/**
+ * @test
+ * @bug 8301260
+ * @summary Check that secureValidation system property works correctly
+ * @library /test/lib
+ * @run main/othervm -Dorg.jcp.xml.dsig.secureValidation=false SecureValidationSystemProperty
+ */
```

Comment:
the jtreg syntax seems to be repeated here.

Reply:
I believe this is now the preferred way to run multiple tests, so that all tests are run even if prior ones fail. It would be nice to establish a best practice on this and update the Developer's Guide. @rhalade is that something you could work with @JesperIRL on?

Reply:
Yes, we will work to get it updated in Developer's guide.